### ISSUE TITLE
Will only scrape reviews of 1 title

### ISSUE THREAD

**Issue by JosedeKruif** [maria-antoniak/goodreads-scraper#9]
Thank you for this very useful notebook. The code for fetching book titles works. The code for fetching reviews starts up, but opens only the reviews for 1 title. After that, reviews of the next title are not fetched. No idea what goes wrong since there is no feedback on errors.
Also, it might be nice to include pandas into the requirements.tx.t

**Comment by maria-antoniak** [comment]
Hi José! Thanks for your feedback. Could you provide some example book IDs so that I can test the problem?

**Comment by JosedeKruif** [comment]
Hi Maria, I am testing the stuff using the example files. Scraping started out with "Pride and Predjudice" and then stopped.  It seems I now found the culprit. Safety measures of my organization prevented "automated" writing in directory by Jupyter. In the mean time.  I was able to perform some further testing. Good news is: the python script works fine when not in Jupyter environment. I will  report some further observations in another issue.
